docs/resources/ges_graph.md

```diff
@@ -0,0 +1,250 @@
+---
+subcategory: "Graph Engine Service (GES)"
+---
+
+# huaweicloud_ges_graph
+
+Manages a GES graph resource within HuaweiCloud.  
+
+## Example Usage
+
+```hcl
+variable "vpc_id" {}
+variable "subnet_id" {}
+variable "secgroup_id" {}
+  
+resource "huaweicloud_ges_graph" "test" {
+  name                  = "demo"
+  graph_size_type_index = "1"
+  arch                  = "x86_64"
+  vpc_id                = var.vpc_id
+  subnet_id             = var.subnet_id
+  security_group_id     = var.secgroup_id
+  crypt_algorithm       = "generalCipher"
+  enable_https          = false
+
+  tags = {
+    key = "val"
+    foo = "bar"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) The graph name.  
+  The name must start with a letter and contains 4 to 50 characters consisting of letters,
+  digits, hyphens (-), and underscores (_). It cannot contain special characters.
+
+  Changing this parameter will create a new resource.
+
+* `graph_size_type_index` - (Required, String) Graph size type index.  
+  Value options are as follows:
+   + **0**: indicates 10 thousand edges.
+   + **1**: indicates 1 million edges.
+   + **2**: indicates 10 million edges.
+   + **3**: indicates 100 million edges.
+   + **4**: indicates 1 billion edges.
+   + **5**: indicates 10 billion edges.
+   + **6**: indicates the database edition.
+   + **401**: indicates 1 billion enhanced edges.
+
+* `vpc_id` - (Required, String, ForceNew) The VPC ID.
+
+  Changing this parameter will create a new resource.
+
+* `subnet_id` - (Required, String, ForceNew) The subnet ID.
+
+  Changing this parameter will create a new resource.
+
+* `security_group_id` - (Required, String, ForceNew) The security group ID.
+
+  Changing this parameter will create a new resource.
+
+* `crypt_algorithm` - (Required, String, ForceNew) Graph instance cryptography algorithm.  
+  Value options are as follows:
+    + **generalCipher**: Chinese cryptographic algorithm.
+    + **SMcompatible**: Commercial cryptography algorithm (compatible with international ones).
+
+  Changing this parameter will create a new resource.
+
+* `enable_https` - (Required, Bool, ForceNew) Whether to enable the security mode. This mode may damage GES performance greatly.
```

Comment:
how about make this optional, and put it with other bool type parameters

Reply:
cannot, enable this mode just depends on this one parameter.